backend/main/server/app.go

```diff
@@ -1875,3 +1865,21 @@ func (a *App) validateTimestamp(timestamp string, expiry int) error {
 	}
 	return nil
 }
+
+func (a *App) processSnapshotStatus(s *models.Strategy, p *models.Proposal) error {
+	var processing = "processing"
+	if s.Contract.Name != nil && p.Snapshot_status == &processing {
+		snapshotResponse, err := a.SnapshotClient.GetSnapshotStatus(s.Contract)
+		if err != nil {
+			return err
+		}
+
+		p.Snapshot_status = &snapshotResponse.Data.Status
+		p.Block_height = snapshotResponse.Data.BlockHeight
```

Comment:
blockheight doesnt need to be added, is already on proposal

Reply:
removed